docusaurus/docs/how-to-guides/panel-plugins/options-migration-handler.md

```diff
@@ -0,0 +1,151 @@
+---
+id: migration-handler-for-panels
```

Comment:
not sure how to structure this but I was thinking on a generic page with general information about plugin migrations (e.g. when to use it) and then split in different pages:

 - Migrations for panel plugins
 - Migration for frontend datasources
 - Migration for backend datasources
 - (Maybe in the future?) Migration for apps

I guess this is fine for the first guide but we may need to restructure?

Reply:
I think it is best to keep these migrations guide under each plugin type and maybe we can create some phantom menu entries for a dedicated migrations section.

if i were a dev I'd look for information about panels under the panels section.